src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final String ROW_SUPPLY = "supply";
+    static final String ROW_BUY = "buy";
+    static final String ROW_RESULT = "result";
+    static final String SEPARATOR = ",";
+    static final int RESET_DATA = 0;
+    static final int INDEX_FIRST = 0;
+    static final int INDEX_SECOND = 1;
+    private int supply = 0;
+    private int buy = 0;
+    private String[] result;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method should contain three calls of other methods: readFile() - returns content, generateReport() -(you pass contents here), writeToFile()-(you pass report and filename here ), this will be a good code decomposition
example:

```
        String data = readFile(fromFileName);
        String report = generateReport(data);
        writeToFile(toFileName, report);
```